src/lib/openapi/spec/api-token-schema.ts

```diff
@@ -24,57 +36,57 @@ export const apiTokenSchema = {
         type: {
             type: 'string',
             enum: Object.values(ApiTokenType),
+            description: 'The type of API token',
+            example: 'client',
         },
         environment: {
             type: 'string',
+            description:
+                'The environment the token has access to. * if it has access to all environments.',
+            example: 'development',
         },
         project: {
             type: 'string',
+            description: 'The project this token belongs to.',
+            example: 'developerexperience',
         },
         projects: {
             type: 'array',
+            description:
+                'The list of projects this token has access to. If the token has access to specific projects they will be listed here. If the token has access to all projects it will be represented as [`*`]',
```

Comment:
Shouldn't those quotes and brackets be swapped?

```suggestion
                'The list of projects this token has access to. If the token has access to specific projects they will be listed here. If the token has access to all projects it will be represented as `[*]`',
```